<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>1161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>1161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCargarCSV">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Cargar CSV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGraficarCSV">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Graficar CSV</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblGraficaCSV">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gráfica CSV</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTituloCsv">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Análisis de CSV</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblvariable">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Variable:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblEstadoCSV">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Estado:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLimpiarGrafica">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>690</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExportarGrafica">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>690</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Exportar Gráfica</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>560</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Variable seleccionada:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblrtaestadoCSV">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>530</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>_____________</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblrtavariableCSV">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>550</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>_________________</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnvolverCSV">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>750</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>VOLVER</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablaCSV">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>371</width>
     <height>161</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="lbldatosCSV">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>480</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DATOS CSV</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidcolumnas">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <item>
    <property name="text">
     <string>Columna 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columna 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columna 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columna 4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>220</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>220</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidcolumnas2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <item>
    <property name="text">
     <string>Columna 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columna 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columna 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columna 4</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="listWidcolumnas4">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <item>
    <property name="text">
     <string>Columna 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columna 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columna 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columna 4</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="listWidcolumnas3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <item>
    <property name="text">
     <string>Columna 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columna 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columna 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Columna 4</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
